mlrun/projects/pipelines.py

```diff
@@ -880,6 +886,7 @@ def load_and_run(
     init_git: bool = None,
     subpath: str = None,
     clone: bool = False,
+    save: bool = True,
```

Comment:
IMO `load_and_run` should be documented, for every param and all its options it supports as this will only grow and grow and we better have the flows its supports documented.